docs/faq.rst

```diff
@@ -13,7 +13,7 @@ Yes, the `deduplication`_ technique used by
 Also, we have optional simple sparse file support for extract.
 
 If you use non-snapshotting backup tools like Borg to back up virtual machines,
-then these should be turned off for doing so. Backing up live VMs this way can (and will)
+then the VMs should be turned off before doing so. Backing up live VMs can (and will)
```

Comment:
Hmm, I am not a native english speaker, but doesn't this change semantics or at least create an ambiguity?

be turned off for doing so == machine is turned off while backup is running.

be turned off before doing so == machine is turned off before backup is started, but unclear if it can be turned on after it is started.